### PR TITLE
Validation fixes html tags

### PR DESCRIPTION
**Pull request by chrisms86**

### CODE REVIEW

**Comment by billmath** [comment]
Docs Build status updates of commit _[b248b0c](https://github.com/chrisms86/azure-docs-powershell-azuread/commits/b248b0cb7bd74ceeb6caa16fb496354b468af7be)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-conceptual/azureadps-2.0-preview/new-user-sample.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/new-user-sample.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/new-user-sample?view=azureadps-2.0-preview&branch=pr-en-us-822) (azureadps-2.0-preview) | [Details](#user-content-b104d989d0eb91510c44328c0450d70c-14bad79b)
[docs-conceptual/azureadps-2.0-preview/signing-in-service-principal.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/signing-in-service-principal.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/signing-in-service-principal?view=azureadps-2.0-preview&branch=pr-en-us-822) (azureadps-2.0-preview) | [Details](#user-content-e89e876d47802b559cc46f3708607e18-14bad79b)
[docs-conceptual/azureadps-2.0-preview/using-extension-attributes-sample.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/using-extension-attributes-sample.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/using-extension-attributes-sample?view=azureadps-2.0-preview&branch=pr-en-us-822) (azureadps-2.0-preview) | [Details](#user-content-c3ff527067d58cdb65e89ece8f1b51c3-14bad79b)
[docs-conceptual/azureadps-2.0/new-user-sample.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/new-user-sample.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/new-user-sample?view=azureadps-2.0&branch=pr-en-us-822) (azureadps-2.0) | [Details](#user-content-1b7609d9a97ce171341f8c8168180496-14bad79b)
[docs-conceptual/azureadps-2.0/signing-in-service-principal.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/signing-in-service-principal.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/signing-in-service-principal?view=azureadps-2.0&branch=pr-en-us-822) (azureadps-2.0) | [Details](#user-content-3325de5cb1416715f8143f00506b22ea-14bad79b)
[azureadps-2.0/AzureAD/New-AzureADServiceAppRoleAssignment.md](https://github.com/chrisms86/azure-docs-powershell-azuread/blob/wb-0511-adpa-cs/azureadps-2.0/AzureAD/New-AzureADServiceAppRoleAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/new-azureadserviceapproleassignment?view=azureadps-2.0&branch=pr-en-us-822) (azureadps-2.0) |
[docs-conceptual/azureadps-1.0/install-msonlinev1.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-1.0/install-msonlinev1.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/install-msonlinev1?view=azureadps-1.0&branch=pr-en-us-822) (azureadps-1.0) |

<a id="b104d989d0eb91510c44328c0450d70c-14bad79b"></a>
### [docs-conceptual/azureadps-2.0-preview/new-user-sample.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/new-user-sample.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="e89e876d47802b559cc46f3708607e18-14bad79b"></a>
### [docs-conceptual/azureadps-2.0-preview/signing-in-service-principal.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/signing-in-service-principal.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="c3ff527067d58cdb65e89ece8f1b51c3-14bad79b"></a>
### [docs-conceptual/azureadps-2.0-preview/using-extension-attributes-sample.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/using-extension-attributes-sample.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="1b7609d9a97ce171341f8c8168180496-14bad79b"></a>
### [docs-conceptual/azureadps-2.0/new-user-sample.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/new-user-sample.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

<a id="3325de5cb1416715f8143f00506b22ea-14bad79b"></a>
### [docs-conceptual/azureadps-2.0/signing-in-service-principal.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/signing-in-service-principal.md)
 - **Line 2, Column 1**: **[Suggestion: description-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/ed7d034d-a56c-a713-9d51-730d2ba643e8/202205111730344922-822/BuildReport?accessString=aa2702e48661ba1626e722d93cdc8c296f3a9bb27608195babdf232f71e21755).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by billmath** [comment]
Docs Build status updates of commit _[85c02cf](https://github.com/chrisms86/azure-docs-powershell-azuread/commits/85c02cf33f3e566fe1678d5c711936ef486fcdd8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/New-AzureADServiceAppRoleAssignment.md](https://github.com/chrisms86/azure-docs-powershell-azuread/blob/wb-0511-adpa-cs/azureadps-2.0/AzureAD/New-AzureADServiceAppRoleAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/new-azureadserviceapproleassignment?view=azureadps-2.0&branch=pr-en-us-822) (azureadps-2.0) |
[docs-conceptual/azureadps-1.0/install-msonlinev1.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-1.0/install-msonlinev1.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/install-msonlinev1?view=azureadps-1.0&branch=pr-en-us-822) (azureadps-1.0) |
[docs-conceptual/azureadps-2.0-preview/new-user-sample.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/new-user-sample.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/new-user-sample?view=azureadps-2.0-preview&branch=pr-en-us-822) (azureadps-2.0-preview) |
[docs-conceptual/azureadps-2.0-preview/signing-in-service-principal.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/signing-in-service-principal.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/signing-in-service-principal?view=azureadps-2.0-preview&branch=pr-en-us-822) (azureadps-2.0-preview) |
[docs-conceptual/azureadps-2.0-preview/using-extension-attributes-sample.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/using-extension-attributes-sample.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/using-extension-attributes-sample?view=azureadps-2.0-preview&branch=pr-en-us-822) (azureadps-2.0-preview) |
[docs-conceptual/azureadps-2.0/new-user-sample.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/new-user-sample.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/new-user-sample?view=azureadps-2.0&branch=pr-en-us-822) (azureadps-2.0) |
[docs-conceptual/azureadps-2.0/signing-in-service-principal.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/signing-in-service-principal.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/signing-in-service-principal?view=azureadps-2.0&branch=pr-en-us-822) (azureadps-2.0) |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/ed7d034d-a56c-a713-9d51-730d2ba643e8/202205111850142776-822/BuildReport?accessString=446125a361646db82f98fe4a90e1843e670ef771bbddb55d68874e7f152bbf4d).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by chrisms86** [comment]
**Global effort to fix build validation errors**
@CelesteDG @msewaweru 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Comment by chrisms86** [comment]
@CelesteDG @msewaweru can you please let me know if you need any additional changes or if this is ready to merge? Thank you!

**Comment by msewaweru** [comment]
@chrisms86, apologies for the delay in reviewing this. I will review tomorrow and get back to you. Thanks.

File: docs-conceptual/azureadps-1.0/install-msonlinev1.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Connect-MsolService -AzureEnvironment "AzureGermanyCloud"
 
 This example connects your PowerShell session to the German AzureAD environment.
 
-See [Connect-MsolService] (https://docs.microsoft.com/powershell/module/msonline/connect-msolservice) for more information.
+See [Connect-MsolService] (/powershell/module/msonline/connect-msolservice) for more information.
```
Comment (msewaweru) [inline docs-conceptual/azureadps-1.0/install-msonlinev1.md:89]:
```suggestion
See [Connect-MsolService](/powershell/module/msonline/connect-msolservice) for more information.
```

File: docs-conceptual/azureadps-2.0-preview/new-user-sample.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 services: active-directory
 documentationcenter: ''
 title: 'Create a new user with AzureAD Preview'
+description: "This example shows how you can create a new user in Azure Active Directory using only mandatory parameters or the cmdlet."
```
Comment (msewaweru) [inline docs-conceptual/azureadps-2.0-preview/new-user-sample.md:5]: Did you mean to say `mandatory parameters of the cmdlet`?

**Comment by msewaweru** [comment]
@chrisms86, please take a look at the issues I highlighted.

File: docs-conceptual/azureadps-1.0/install-msonlinev1.md (hunk 1 above)
Comment (chrisms86) [reply docs-conceptual/azureadps-1.0/install-msonlinev1.md]: @msewaweru resolved

**Comment by billmath** [comment]
Docs Build status updates of commit _[edf4452](https://github.com/chrisms86/azure-docs-powershell-azuread/commits/edf4452422998e9e10a7342c53f3919219f9371e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/New-AzureADServiceAppRoleAssignment.md](https://github.com/chrisms86/azure-docs-powershell-azuread/blob/wb-0511-adpa-cs/azureadps-2.0/AzureAD/New-AzureADServiceAppRoleAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/new-azureadserviceapproleassignment?view=azureadps-2.0&branch=pr-en-us-822) (azureadps-2.0) |
[docs-conceptual/azureadps-1.0/install-msonlinev1.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-1.0/install-msonlinev1.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/install-msonlinev1?view=azureadps-1.0&branch=pr-en-us-822) (azureadps-1.0) |
[docs-conceptual/azureadps-2.0-preview/new-user-sample.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/new-user-sample.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/new-user-sample?view=azureadps-2.0-preview&branch=pr-en-us-822) (azureadps-2.0-preview) |
[docs-conceptual/azureadps-2.0-preview/signing-in-service-principal.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/signing-in-service-principal.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/signing-in-service-principal?view=azureadps-2.0-preview&branch=pr-en-us-822) (azureadps-2.0-preview) |
[docs-conceptual/azureadps-2.0-preview/using-extension-attributes-sample.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/using-extension-attributes-sample.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/using-extension-attributes-sample?view=azureadps-2.0-preview&branch=pr-en-us-822) (azureadps-2.0-preview) |
[docs-conceptual/azureadps-2.0/new-user-sample.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/new-user-sample.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/new-user-sample?view=azureadps-2.0&branch=pr-en-us-822) (azureadps-2.0) |
[docs-conceptual/azureadps-2.0/signing-in-service-principal.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/signing-in-service-principal.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/signing-in-service-principal?view=azureadps-2.0&branch=pr-en-us-822) (azureadps-2.0) |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/ed7d034d-a56c-a713-9d51-730d2ba643e8/202205181754143582-822/BuildReport?accessString=2bf034fb1704eb7a1fde105c4cd7fd45e86e0a6a628a9b39de10913599584c79).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by msewaweru — APPROVED** [review]

**Comment by msewaweru** [comment]
LGTM! Thanks @chrisms86.

**Comment by msewaweru** [comment]
@CelesteDG, this is ready to merge.

**Pull request merged**